pkgs/top-level/all-packages.nix

```diff
@@ -15899,6 +15899,8 @@ in
   parity-beta = res.altcoins.parity-beta;
   parity-ui = res.altcoins.parity-ui;
 
+  polkadot = res.altcoins.polkadot;
```

Comment:
Let's not have it exposed as both pkgs.altcoins.polkadot and pkgs.polkadot

Reply:
So, let's not have expose parity/parity-beta/parity-ui too? In my opinion agreement around exports here should be the same.

Reply:
I didn't even notice that, you're probably right.